tests/test_integration/test_dace_parsing.py

```diff
@@ -0,0 +1,248 @@
+import re
+
+import hypothesis.strategies as hyp_st
+import numpy as np
+import pytest
+
+from gt4py import gtscript
+from gt4py.gtscript import PARALLEL, computation, interval
+
+
+dace = pytest.importorskip("dace")
+
+
+@pytest.fixture
+def dace_stencil():
+    @gtscript.stencil(backend="gtc:dace")
+    def defn(inp: gtscript.Field[np.float64], outp: gtscript.Field[np.float64]):
+        with computation(PARALLEL), interval(...):
+            outp = inp  # noqa F841: local variable 'outp' is assigned to but never used
+
+    return defn
+
+
+def tuple_st(min_value, max_value):
+    return hyp_st.tuples(
+        hyp_st.integers(min_value=min_value, max_value=max_value),
+        hyp_st.integers(min_value=min_value, max_value=max_value),
+        hyp_st.integers(min_value=min_value, max_value=max_value),
+    )
+
+
+def test_basic():
+    @gtscript.stencil(backend="gtc:dace")
+    def defn(outp: gtscript.Field[np.float64], par: np.float64):
+        with computation(PARALLEL), interval(...):
+            outp = par  # noqa F841: local variable 'outp' is assigned to but never used
+
+    outp = np.zeros(dtype=np.float64, shape=(10, 10, 10))
```

Comment:
Should these tests use both `numpy` storages as well as `gt4py.storage.Storage`?

Reply:
I added one test that uses storages instead. DaCe doesn't like that storages are a view of the underlying ndarray, so we have to override `TypeError: Passing a numpy view (e.g., sub-array or "A.T") to DaCe programs is not allowed in order to retain analyzability. Please make a copy with "numpy.copy(...)". If you know what you are doing, you can override this error in the configuration by setting compiler.allow_view_arguments to True.`

Reply:
You can resolve that by overriding the configuration. We will improve the condition to only occur on truly unsafe view arguments (i.e., mismatching strides, as opposed to the offsetted arrays which we are seeing here)

Reply:
> `TypeError: Passing a numpy view (e.g., sub-array or "A.T") to DaCe programs is not allowed in order to retain analyzability. Please make a copy with "numpy.copy(...)". If you know what you are doing, you can override this error in the configuration by setting compiler.allow_view_arguments to True.`

Ah, I understand, but it concerns me and points to a code smell/issue that needs to be addressed.

`numpy.ndarray` are not fully supported as arguments to stencils (since they lack metadata like `default_origin`, `mask`, `layout`, etc.), so stating "use numpy arrays with dace" essentially amounts to throwing away the carefully-designed launch API of gt4py and relying solely on frozen stencils (which are derived from the `StencilObject`s). In my opinion, frozen stencils should be employed as an optimization over the regular stencil interface and hidden in application utility layers, not used as a top-level interface.

I have one idea: Can this exchange from `Storage` to `ndarray` (probably just taking the `.data` attribute) be done in the `DaceStencilObject`?

cc: @havogt @tbennun

Reply:
@tbennun it seems like our comments stepped on each other. Glad to hear it there's a way around it.

Reply:
What Tal suggests is exactly what line 253 in test_dace_parsing.py does. I will change all the tests to work with storages, there's no point testing it with numpy at all.